src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -81,7 +85,7 @@ private GitHubUserCache(Docset docset, string token)
         public static async Task<GitHubUserCache> Create(Docset docset, string token)
         {
             var result = new GitHubUserCache(docset, token);
-            await result.ReadCacheFile();
+            await result.ReadCacheFiles();
             return result;
         }
```

Comment:
https://github.com/dotnet/docfx/blob/af768136d161398b3f6c909e368328df6569adad/src/docfx/lib/github/GitHubUserCache.cs#L71

We may treat url and file path the same, `GetFileRestorePath` does not need a output `(fromUrl)`

Reply:
Want to write to GitHubUserCache directly if it points to a local file, then keep `fromUrl` can save some duplicated call to `HrefUtility.IsHttpHref`.